rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -2379,6 +2379,17 @@ public Coin getLockingCap() {
         return this.provider.getLockingCap();
     }
 
+    /**
+     * Returns the redeemScript of the current active federation
+     * @return Returns the redeemScript of the current active federation
+     */
+    public Script getActivePowpegRedeemScript() {
+        if (activations.isActive(ConsensusRule.RSKIP293)) {
+            return getActiveFederation().getRedeemScript();
+        }
+        return null;
```

Comment:
I don't like the idea of returning null by default, this can lead to unexpected NPE somewhere along the line. Maybe return `new Script(new byte[]{})`?

Reply:
I wonder, what if I didn't check the activation here? The activation is already being checked on the `BridgeMethods` which from what I understand is the method through externals will use to get this information.

![image](https://user-images.githubusercontent.com/94651730/151870238-d356a12b-7fb6-4480-bd79-9d9b9294de86.png)

As there is always going to exist an active federation that always has a redeemScript, this method doesn't need to return null. Furthermore making it null safety.

Reply:
You are right, the activation is already checked when calling the method. I guess it doesn't hurt either to have a double check